Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -235,11 +236,12 @@ open class MessageContentCell: MessageCollectionViewCell {
             }
         }
 
+        let avatarPadding = attributes.avatarPosition.padding
         switch attributes.avatarPosition.horizontal {
         case .cellLeading:
-            origin.x = attributes.avatarSize.width + attributes.messageContainerPadding.left
+            origin.x = attributes.avatarSize.width + attributes.messageContainerPadding.left + avatarPadding.horizontal
         case .cellTrailing:
-            origin.x = attributes.frame.width - attributes.avatarSize.width - attributes.messageContainerSize.width - attributes.messageContainerPadding.right
+            origin.x = attributes.frame.width - attributes.avatarSize.width - attributes.messageContainerSize.width - attributes.messageContainerPadding.right - avatarPadding.horizontal
```

Comment:
We use `horizontal` here because we need to offset it the total size to layout the container, right?